config/config.go

```diff
@@ -1312,6 +1364,8 @@ type RemoteWriteConfig struct {
 	// values arbitrarily into the overflow maps of further-down types.
 	HTTPClientConfig HTTPClientConfig `yaml:",inline"`
 
+	RemoteQueueConfig RemoteQueueConfig `yaml:",inline"`
```

Comment:
I wasn't sure either - am happy to leave them as a config struct that isn't "exposed".